app/src/main/res/values/styles.xml

```diff
@@ -3,7 +3,7 @@
     <!-- Base application theme. -->
     <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
         <!-- Customize your theme here. -->
-        <item name="colorPrimary">@color/colorPrimary</item>
+        <item name="colorPrimary">@color/colour_main</item>
```

Comment:
Why don't you change the value of `colorPrimary` in `colors.xml` instead of deleting `colorPrimary` and adding `colour_main`?